### PR TITLE
Mapped-Superclasses with embeddables

### PR DESCRIPTION
**Pull request by bartoszpietrzak1994**
Fixes https://github.com/Sylius/InvoicingPlugin/issues/120

### CODE REVIEW

File: src/Doctrine/Mapping/Driver/DriverChain.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            $this->entityManager->getEventManager()->dispatchEvent('preEmbeddableResolve', $eventArgs);
+        }
+    }
+
+    public function getAllClassNames(): array
+    {
+        return $this->baseMappingDriverChain->getAllClassNames();
+    }
+
+    public function isTransient($className): bool
```
Comment (Tomanhez) [inline src/Doctrine/Mapping/Driver/DriverChain.php]:
```suggestion
    public function isTransient(? $className): bool
```
missing type

File: src/Resources/config/services.xml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
+        <import resource="services/doctrine.xml"/>
```
Comment (Tomanhez) [inline src/Resources/config/services.xml]:
```suggestion
        <import resource="services/doctrine.xml" />
```
missing space

File: src/Doctrine/Mapping/Driver/DriverChain.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    /** @var EntityManager */
+    private $entityManager;
+
+    public function __construct(BaseMappingDriverChain $baseMappingDriverChain, EntityManager $entityManager)
+    {
+        $this->baseMappingDriverChain = $baseMappingDriverChain;
+        $this->entityManager = $entityManager;
+    }
+
+    public function loadMetadataForClass($className, ClassMetadata $metadata): void
```
Comment (Tomanhez) [inline src/Doctrine/Mapping/Driver/DriverChain.php]: missing type
```suggestion
    public function loadMetadataForClass(? $className, ClassMetadata $metadata): void
```

**Comment by pamil** [comment]
Posting it as a regular comment so that this issue gets linked to https://github.com/Sylius/SyliusResourceBundle/pull/89.

**Pull request closed without merging**